### PR TITLE
Added setup for Windows Command Prompt (cmd)

### PR DESCRIPTION
**Pull request by joedf**
Setup for CMD / Windows Command Prompt.

### CODE REVIEW

**Comment by mgunyho** [comment]
Thanks! Could you add an if-else check that the output is not empty. This way e.g. `tere --help` should work correctly (please check that it does, I don't know how cmd handles stderr/stdout).

Note that I will be offline for a couple of weeks starting tomorrow so it will take a while before I can merge this.

**Comment by joedf** [comment]
No worries. Looks like like I forgot to pass on arguments, fixed now.
As for the empty check `IF [%OUTPUT%] == [] goto :EOF`, there's no use for it to be there since the help text is processed. This okay however since `cd` will simply do nothing.

**Comment by joedf** [comment]
Been using it since, works great no issues so far! 👍

**Comment by mgunyho** [comment]
I'm glad to hear that! Just one more question: does `tere --version` or `tere --help` work as expected?

File: README.md [hunk 1]
```diff
@@ -55,20 +55,32 @@ To start using `tere`, follow these steps:
...
-                $tere_path = Join-Path $tere_dir 'tere'
-            }
-            $result = . $tere_path
-            if ($result) {
-                Set-Location $result
-            }
+    function Invoke-Tere() {
+        $tere_dir = '/path/to/tere'
+        if ($isWindows) {
+            $tere_path = Join-Path $tere_dir 'tere.exe'
```
Comment (mgunyho) [inline README.md:61]: While we're here, is this `isWindows` check needed? Wouldn't it work without the exe suffix?

File: README.md [hunk 2]
```diff
@@ -55,20 +55,32 @@ To start using `tere`, follow these steps:
...
+        }
+        $result = . $tere_path
+        if ($result) {
+            Set-Location $result
+        }
+    }
+    Set-Alias tere Invoke-Tere  
+    ``` 
+
+    For Windows Command Prompt, put this in a batch script file called `tere.bat` in a folder included in your `PATH` environment variable such as `C:\Windows`:
```
Comment (mgunyho) [inline README.md:74]: If you install via cargo, that will create `tere.exe` under some cargo bin folder that is in PATH by default, right? Does this conflict with that?

File: README.md [hunk 3]
```diff
@@ -55,20 +55,32 @@ To start using `tere`, follow these steps:
...
-        Set-Alias tere Invoke-Tere  
+        else {
+            $tere_path = Join-Path $tere_dir 'tere'
+        }
+        $result = . $tere_path
+        if ($result) {
+            Set-Location $result
+        }
+    }
+    Set-Alias tere Invoke-Tere
```
Comment (superatomic) [inline README.md:71]: There's some trailing whitespace here that should probably be removed:

```suggestion
    Set-Alias tere Invoke-Tere
```

**Comment by joedf** [comment]
> I'm glad to hear that! Just one more question: does `tere --version` or `tere --help` work as expected?

Yes, they do. Just tested again to be sure.

File: README.md (hunk 1 above)
Comment (joedf) [reply README.md:61]: I am actually not sure about this... I tried it with and without `.exe` and it worked for me. Albeit I simplified to just this:
```ps
function InvokeTere() {
  $tere = 'C:/path/to/location/of/tere.exe'
  $command = . $tere
  Set-Location $command
}
Set-Alias tere InvokeTere
```

File: README.md (hunk 2 above)
Comment (joedf) [reply README.md:74]: I have not tried it `cargo install`, but I don't why it wouldn't be so.

File: README.md (hunk 1 above)
Comment (mgunyho) [reply README.md:61]: Okay, thanks! Maybe we can leave it as it is now, I'll do some further testing later.

**Comment by mgunyho** [comment]
Thanks a lot! I'll rebase and then merge this.

File: README.md (hunk 1 above)
Comment (joedf) [reply README.md:67]: I've also noticed that the Powershell config given doesn't support arguments... so no support for `--version` or `--help`... I don't really use powershell so I'm not sure how to fix it.

Comment (mgunyho) [reply README.md:67]: Ah, good point! Don't worry about it, I'll figure it out.

File: README.md (hunk 2 above)
Comment (joedf) [reply README.md:80]: Yes, I just did `cargo build --release`, then tested and copied the `tere.exe` file.
I am just slowly getting into rust actually. 😅

**Pull request merged**